docs/source/index.rst

```diff
@@ -151,10 +151,17 @@ messy situations in everyday problems.
    dataframe.rst
    delayed.rst
    futures.rst
+   how-to/connect-to-remote-data.rst
```

Comment:
This should stay in the "How to" section.

Reply:
I'll remove this for now, although I think that it's a mistake to not keep this top-level.  I think that "How do I access data?" should be a question that someone should be able to easily answer when looking at the docs.